engine/src/flutter/shell/platform/linux/fl_engine.cc

```diff
@@ -379,6 +379,19 @@ static void fl_engine_platform_message_cb(const FlutterPlatformMessage* message,
   }
 }
 
+// Called when a log message is requested to be logged by the engine.
+static void fl_engine_log_message_cb(const char* tag,
+                                     const char* message,
+                                     void* user_data) {
+  if (g_strcmp0(tag, "") != 0) {
```

Comment:
For reference `g_strcmp0` treats `NULL` as the empty string. See docs here:
https://docs.gtk.org/glib/func.strcmp0.html